### ISSUE TITLE
NullReference is thrown in Binder.GetValue("section", defaultValue)

### ISSUE THREAD

**Issue by MihailsKuzmins** [MyNihongo/Moq.Microsoft.Configuration#23]
```cs
MockConfiguration
	.SetupConfiguration()
	.Returns(new
	{
		Scanner = new
		{
			PortNumber = portNumber
		}
	});

var value = MockConfiguration.Object.GetValue("NotDefined", "defaultValue");
```

**Comment by my-nihongo** [comment]
Fixed in #24

**Comment by my-nihongo** [comment]
Fixed only for the RootNode.
Not fixed when
```cs
var section = MockConfiguration.Object.GetSection("Exists");
var value = section.GetValue<int>("NotExists")l
```

**Comment by my-nihongo** [comment]
Fixed in #25
